docs/dev/serialization.md

```diff
@@ -79,29 +79,29 @@ If the returned object has a `_from_json_dict_` attribute, it is called instead.
 
 ## Adding a new serializable value
 
-All of Cirq's public classes should be serializable.
+All of Cirq's public classes should be serializable. Public classes are the ones that can be found in the Cirq module top level 
+namespaces, i.e. `cirq.*`, `cirq_google.*`, `cirq_aqt.*`, etc, (see [Cirq modules](./modules.md) for setting up JSON serialization for a module).  
 This is enforced by the `test_json_test_data_coverage` test in
-`cirq/protocols/json_serialization_test.py`, which iterates over cirq's API looking for types
-with no associated json test data.
+`cirq-core/cirq/protocols/json_serialization_test.py`, which iterates over cirq's API 
+looking for types with no associated json test data.
 
-There are several steps needed to get a object serializing, deserializing, and
-passing tests:
+There are several steps needed to support an object's serialization and deserialization, 
+and pass `cirq-core/cirq/protocols/json_serialization_test.py`:
 
 1. The object should have a `_json_dict_` method that returns a dictionary
 containing a `"cirq_type"` key as well as keys for each of the value's
-attributes.
-If these keys do not match the names of the class' initializer arguments, a
-`_from_json_dict_` class method must also be defined.
+attributes. If these keys do not match the names of the class' initializer 
+arguments, a `_from_json_dict_` class method must also be defined.
 Typically the `"cirq_type"` will be the name of your class.
 
-2. Add an entry to the big hardcoded dictionary in `cirq/protocols/json.py`,
-mapping the cirq_type string you chose to the class.
-You can also map the key to a helper method that returns the class (important
-for backwards compatibility if e.g. a class is later replaced by another one).
-After doing this, `cirq.to_json` and `cirq.read_json` should start working for
-your object.
+2. In `class_resolver_dictionary` within the packages's `json_resolver_cache.py` file,
+for each serializable class, map keys matching the `cirq_type` to the 
```

Comment:
this is an imperative tense: "map keys to stuff". Instead of "the dictionary in the file maps keys to stuff" or "the dictionary in the file should map keys to stuff" like the other file/comment. Again: I suggest you pick a form and stick with it.